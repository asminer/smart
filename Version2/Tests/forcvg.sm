
/*
    $Id$
 */
	
/*
    Tests a for - converge nesting

*/

# Verbose true
# ConvergePrecision 1e-7
	
for (int i in {1..10}) {
  converge {
    real x[i] guess 100*i; 
    real x[i] := sqrt(i*x[i]);
  }
  print("x[", i, "] converged to ", x[i], "\n"); 
}

