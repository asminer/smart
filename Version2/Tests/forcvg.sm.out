x[1] converged to 1
x[2] converged to 2
x[3] converged to 3
x[4] converged to 4
x[5] converged to 5
x[6] converged to 6
x[7] converged to 7
x[8] converged to 8
x[9] converged to 9
x[10] converged to 10
